chapters/3-measurement.tex

```diff
@@ -49,74 +60,91 @@
 
 \section{Creating a data map}
 
-In most projects, more than one data source is needed to answer the research question.
+In most projects, more than one data source is needed to answer a research question.
 These could be data from multiple survey rounds,
-data acquired from different partners (such as administrative data, implementation data, sensor data),
-web scraping,
+data acquired from different partners 
+(such as administrative data, implementation data, sensor data),
+technological tools like satellite imagery or web scraping,
 or complex combinations of these and other sources.\sidenote{
   See \citet{kondylis2018speed} for an example.}
-However your study is structured, you need to know how to link data from all sources
+However your study data is structured, 
+you need to know how to link data from all sources
 and analyze the relationships between the units that appear in them
-to answer all your research questions.
+to answer your research questions.
 You might think that you are able to keep all the relevant details in your head,
 but your whole research team is unlikely to have the same understanding,
 at all times, of all the datasets required.
 The only way to make sure that the full team shares the same understanding
-is to create a \textbf{data map}\index{Data map}.\sidenote{
+is to create a \textbf{data map}\index{data map}.\sidenote{
 	\url{https://dimewiki.worldbank.org/Data\_Map}}
 DIME's data map template has three components:
-one \textit{data linkage table},\index{Data linkage table}
-one or several \textit{master datasets}\index{Master datasets}
-and one or several \textit{data flowcharts}.\index{Data flowchart}
-
-A \textbf{data linkage table}\sidenote{
+one \textit{data linkage table},\index{data linkage table}
+one or several \textit{master datasets}\index{master datasets}
+and one or several \textit{data flowcharts}.\index{data flowchart}
+
+Each component of the data map
+that you create for your project
+will contribute to understanding
+what the data is about and how to use it.
+First, you will create a \textbf{data linkage table}\sidenote{
 	\url{https://dimewiki.worldbank.org/Data\_Linkage\_Table}}
-lists all the datasets that will be used in the project.
-Its most important function is to indicate
-how all those datasets can be be linked when
-combining information from multiple data sources.
-Each \textbf{master dataset}\sidenote{
+listing all the raw datasets that will be used in the project,
+what data sources they are created from,
+and how they relate to each other.
+Second, for each \textbf{unit of observation}\sidenote{
+	\url{https://dimewiki.worldbank.org/Unit\_of\_Observation}} 
+in the data linkage table,
+you will maintain a \textbf{master dataset}\sidenote{
```

Comment:
@bbdaniels This is better language, but it misses some cases for which master datasets need to be created. The data linkage table only list your units of data acquisition, but you might have more unit of observations that should have master datasets. Let say you collect data on household but your level of analysis are household members, then household members is not in the data linkage table, but it should have a master dataset. same is possible for sampling and assignment. Can we change this so it deos not seem to limited it to the data linkage table, but keep the nicer language?

Reply:
Is this good: 5bc3aa415c2fb166b462b4fa2a68b491ce62e984